src/style.css

```diff
@@ -0,0 +1,98 @@
+:root {
+  --darkColor: rgba(61, 78, 97, 0.1);
+  --lightColor: #3d4e61;
+  --borderColor: #e1e7ed;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  font-family: Avenir, sans-serif;
```

Comment:
it's better to move this rule to `html`/`body` tag. We should use only `box-sizing: border-box;` for `*` selector